mtp_api/apps/disbursement/tests/test_views.py

```diff
@@ -0,0 +1,135 @@
+from django.core.urlresolvers import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from disbursement.constants import (
+    DISBURSEMENT_RESOLUTION, DISBURSEMENT_METHOD, LOG_ACTIONS
+)
+from disbursement.models import Disbursement, Recipient, Log
+from core.tests.utils import make_test_users
+from mtp_auth.tests.utils import AuthTestCaseMixin
+from prison.models import Prison
+
+
+class CreateDisbursementTestCase(AuthTestCaseMixin, APITestCase):
+    fixtures = ['initial_types.json', 'test_prisons.json', 'initial_groups.json']
+
+    def setUp(self):
+        super().setUp()
+        self.prison_clerks, _, self.bank_admins, _, _, _ = make_test_users()
+
+    def test_permissions_required(self):
```

Comment:
i think it'd be nice to have a test for a prison clerk failing to make a disbursement for someone not in their prison… or do you feel that we don't need to force that rule?

Reply:
I guess probably should for consistency.